src/bitmessagecli.py

```diff
@@ -136,24 +136,24 @@ def apiInit(apiEnabled):
         return True
 
     else:  # API information was not present.
-        print ('\n     ' + str(keysPath) + ' not properly configured!\n')
+        print('\n     ' + str(keysPath) + ' not properly configured!\n')
         uInput = userInput("Would you like to do this now, (Y)es or (N)o?").lower()
 
         if uInput == "y":  # User said yes, initalize the api by writing these values to the keys.dat file
-            print (' ')
+            print(' ')
 
             apiUsr = userInput("API Username")
             apiPwd = userInput("API Password")
             apiPort = userInput("API Port")
-            apiEnabled = userInput("API Enabled? (True) or (False)").lower()
-            daemon = userInput("Daemon mode Enabled? (True) or (False)").lower()
+            apiEnabled = userInput("API Enabled?(True) or(False)").lower()
+            daemon = userInput("Daemon mode Enabled?(True) or(False)").lower()
 
             if (daemon != 'true' and daemon != 'false'):
-                print ('\n     Invalid Entry for Daemon.\n')
+                print('\n     Invalid Entry for Daemon.\n')
                 uInput = 1
```

Comment:
Maybe
```
uInput = "1"
```

Needs to be tested, but fixes code quality complaint.

Reply:
Updated